webapp/src/main/java/com/box/l10n/mojito/service/tm/importer/TextUnitBatchImporterService.java

```diff
@@ -274,6 +284,12 @@ void importTextUnitsOfLocaleAndAsset(
                         .getTmTextUnitVariant()
                         .getId();
 
+                this.integrityCheckFailureService.updateIntegrityCheckFailures(
```

Comment:
We should have this functionality behind a configuration parameter, that we can easily enable/disable in case of any issues.